inst/templates/tiltWorkflows.Rmd

```diff
@@ -1,236 +1,207 @@
 knitr::opts_chunk$set(
   collapse = TRUE,
-  comment = "#>",
-  # FIXME when this file become reproducible
-  eval = FALSE
+  comment = "#>"
 )
 ```
 
-This file uses the output from the tiltIndicatorBefore package as the inputs to
-the tiltIndicator and tiltIndicatorAfter packages. This script uses the CSV
-files from the tiltIndicatorBefore package to create the final output of the
-four indicators.
+This workflow creates the output of all tilt indicators.
 
 ## Setup
 
 ```{r}
 library(dplyr, warn.conflicts = FALSE)
 library(readr, warn.conflicts = FALSE)
-library(tidyr, warn.conflicts = FALSE)
-# devtools::install_github("2DegreesInvesting/tiltIndicatorAfter")
-library(tiltIndicatorAfter)
-```
-
-## tiltIndicatorAfter data common for all indicators
-
-`ep_companies`, `ei_activities_overview`, `mapper_ep_ei`, `ei_input_data`, and
-`isic_4digit_name` are the five datasets which are used as the input for the
-tiltIndicatorAfter package. They all are the output from tiltIndicatorBefore
-package. These five datasets are used for adding additional data to all the four
-indicators (Emissions profile, Emissions profile upstream, Sector profile,
-Sector profile upstream).
-
-```{r}
-# ep_companies
-europages_campanies <- read_csv("/path/ep_companies.csv") |>
-  select("company_name", "country", "company_city", "postcode", "address", "main_activity", "companies_id")
-
-# ecoinvent_activities
-ei_activities_ov <- read_csv("/path/ei_activities_overview.csv")
-
-# matches_mapper
-mapper_ep_ei <- read_csv("/path/mapper_ep_ei.csv")
+library(fs)
+library(tiltWorkflows)
 
-# ecoinvent_inputs
-ei_input <- read_csv("/path/ei_input_data.csv") |>
-  select("input_activity_uuid_product_uuid", "exchange_name", "exchange_unit_name") |>
-  distinct()
+# Helper
+tilt_path <- function(...) {
+  path_home("Downloads", "tilt", ...)
+}
 
-# isic_4digit_name
-isic_4digit_name <- read_csv("/path/isic_4digit_name.csv")
+dir_create(tilt_path())
+options(width = 500)
 ```
 
-## Emissions profile
-
-### Load Data
+## Data
 
-The input data of Emissions_profile which comes from tiltIndicatorBefore package
-becomes the input for the tiltIndicator package.
+Here we use toy datasets that you should replace with your real data.
 
 ```{r}
-# emissions_profile companies
-ep_company <- read_csv("/path/emissions_profile_any_companies_ecoinvent.csv")
+europages_campanies <- tiltIndicatorAfter::ep_companies |>
+  # TODO: Ask why the toy data has more columns than we need
+  select(
+    "company_name",
+    "country",
+    "company_city",
+    "postcode",
+    "address",
+    "main_activity",
+    "companies_id"
+  )
+
+ecoinvent_activities <- tiltIndicatorAfter::ecoinvent_activities
+
+ecoinvent_europages <- tiltIndicatorAfter::matches_mapper
+
+ecoinvent_inputs <- tiltIndicatorAfter::ecoinvent_inputs |>
+  # TODO: Ask why the toy data has more columns than we need
+  select(
+    "input_activity_uuid_product_uuid",
+    "exchange_name",
+    "exchange_unit_name"
+  ) |>
+  distinct()
```

Comment:
@kalashsinghal 
Can we remove the other columns (if any) from the toy data, and also store only distinct rows?

Reply:
@maurolepore Yes we can do that. But the user will pass all the other columns as well with real data which should not happen. Do you see any other solution to this problem?

FYI: All the other columns are needed in tiltIndicatorBefore code for unit testing. Hence, those columns can'r be removed in tiltIndicatorBefore code.

Reply:
I think we can do two things:

1. If the `profile*()` functions only use some columns, then they should select the useful columns. Once `profile*()` functions do that, then we can remove the columns from this file.

2. If tiltIndicatorBefore needs some columns that the user does not need, then those columns should stay in a testing-version of those datasets but removed from the "released" version of those datasets. This is optional if do the item 1. above.

Reply:
@maurolepore  Please note that only `profile_emissions_upstream` and `profile_sector_upstream` functions use the `ecoinvent_inputs` file. Yes we can select the columns in these functions. Also, we can definitely think about testing and released versions of these datasets. But I guess thats something for later. Thanks for creating separate issue!